### PR TITLE
feat: remove ColorRGBA64 instance exports

### PR DESCRIPTION
**Pull request by nicholasrice**
# Description
BREAKING CHANGE: Removes named `ColorRGBA64` exports

There is a significant terser bug preventing tree-shaking of class instances. This means that all named colors are being bundled regardless of if they are used or not. Re-factoring to a function to retrieve a color by name allows us to properly tree-shake named colors when not explicitly imported, saving roughy 4kb of parsed size and 1kb gzipped.

## Motivation & context
Reducing bundle size

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [x] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [x] This change causes current functionality to break.

## Process & policy checklist

- [x] I have added tests for my changes.
- [ ] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-colors/src/color-converters.ts [hunk 1]
```diff
@@ -16,18 +16,6 @@ import { degreesToRadians, radiansToDegrees } from "./math-utilities";
...
- * Get the relative luminance of a color.
- * Adjusts the color to sRGB space, which is necessary for the WCAG contrast spec.
- * The alpha channel of the input is ignored.
- * @param rgb The input color
- *
- * @deprecated Use rgbToRelativeLuminance instead.
- */
-export function rgbToLuminance(rgb: ColorRGBA64): number {
-    return rgbToRelativeLuminance(rgb);
-}
```
Comment (janechu) [inline packages/fast-colors/src/color-converters.ts]: should we wait for a breaking change before removing this?

File: packages/fast-colors/src/named-colors.ts [hunk 2]
```diff
@@ -0,0 +1,894 @@
+import { ColorRGBA64Config } from "./color-rgba-64";
+export type NamedColors =
```
Comment (janechu) [inline packages/fast-colors/src/named-colors.ts]: can we get a documentation link to where these colors are defined, i assume mdn has these listed

File: packages/fast-colors/src/color-converters.ts (hunk 1 above)
Comment (nicholasrice) [reply packages/fast-colors/src/color-converters.ts]: this *is* a breaking change

File: packages/fast-colors/src/named-colors.ts (hunk 2 above)
Comment (nicholasrice) [reply packages/fast-colors/src/named-colors.ts]: added

**Pull request merged**